src/routes/communityRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const communityController = require("../controllers/communityController");
+
+const router = express.Router();
+
+// // 유저 로그인 여부 검증
+const { validateToken } = require("../utils/auth.js");
+
+router.post("/", validateToken, communityController.createPost);
```

Comment:
지금 작성하신 엔드포인트 주소는 `POST :3000/community/`인데 이 주소는 커뮤니티를 생성하는 엔드포인트라고 보여지지 포스트를 생성하는 엔드포인트라고 보여지지 않습니다. `포스트를 생성하는 엔드포인트구나` 라고 생각이 들게끔 정확하게 작성해주세요.

또한 주소 마지막에는 `/`를 붙이지 말아주세요.
```suggestion
router.post("", validateToken, communityController.createPost);
```

Reply:
멘토님 저도 이 부분 엔드포인트 짤 때 고민을 했는데 커뮤니티와 관련된 저희 필수 구현요소가 이렇습니다.
- 커뮤니티 피드(전체 게시글) 조회
- 커뮤니티에 글쓰기

그래서 한참 고민하다가 그냥 한가지 community 로 묶는 게 낫다고 생각했고,
CRUD 여부는 GET/POST/DELETE 로 구분 가능하다고 생각하여 단순히 POST :3000/community/ 로만 짰는데
제안해주신 피드백을 고려하면 router.post("", validateToken, communityController.createPost); 로 고치기보다는
router.post("/createpost", validateToken, communityController.createPost); 가 낫지 않을까요?

Reply:
POST Method에 의미가 포함되어 있으니`create`는 빼는 것이 좋겠습니다.

Reply:
수정했습니다! 🫡